### ISSUE TITLE
circleci: failed tests (14058): TestExplainTrace 

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#4990]
The following test appears to have failed:

[#14058](https://circleci.com/gh/cockroachdb/cockroach/14058):

```
E160308 19:24:54.360190 client/txn.go:346  failure aborting transaction: kv/txn_coord_sender.go:405: transaction is already committed or aborted; abort caused by: node unavailable; try another peer
W160308 19:24:54.360328 kv/dist_sender.go:627  failed to invoke AdminSplit [/Table/50/0,/Min): storage/replica_command.go:1386: split at key /Table/50 failed: node unavailable; try another peer
I160308 19:24:54.360388 client/db.go:491  failed batch: storage/replica_command.go:1386: split at key /Table/50 failed: node unavailable; try another peer
E160308 19:24:54.360498 storage/queue.go:440  [split] range=5 [/Table/14-/Max): error: storage/split_queue.go:100: unable to split range=5 [/Table/14-/Max) at key "/Table/50/0": storage/replica_command.go:1386: split at key /Table/50 failed: node unavailable; try another peer
I160308 19:24:54.360901 storage/engine/rocksdb.go:141  closing in-memory rocksdb instance
--- FAIL: TestExplainTrace (0.18s)
panic: runtime error: slice bounds out of range [recovered]
	panic: runtime error: slice bounds out of range [recovered]
	panic: runtime error: slice bounds out of range

goroutine 25856 [running]:
panic(0x16aa2c0, 0xc8200100a0)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
testing.tRunner.func1(0xc820432090)
	/usr/local/go/src/testing/testing.go:467 +0x192
panic(0x16aa2c0, 0xc8200100a0)
	/usr/local/go/src/runtime/panic.go:426 +0x4e9
github.com/cockroachdb/pq.(*conn).errRecover(0xc82023f180, 0xc82031fac8)
	/go/src/github.com/cockroachdb/pq/error.go:482 +0x1bd
panic(0x16aa2c0, 0xc8200100a0)
	/usr/local/go/src/runtime/panic.go:426 +0x4e9
github.com/cockroachdb/pq.parseTs(0x0, 0xc82051c170, 0x7, 0x0, 0x0)
	/go/src/github.com/cockroachdb/pq/encode.go:312 +0x140c
github.com/cockroachdb/pq.textDecode(0xc82023f3a8, 0xc82023f1a6, 0x7, 0x1fa, 0x45a, 0x0, 0x0)
	/go/src/github.com/cockroachdb/pq/encode.go:91 +0x5aa
github.com/cockroachdb/pq.decode(0xc82023f3a8, 0xc82023f1a6, 0x7, 0x1fa, 0xc80000045a, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/pq/encode.go:62 +0x97
github.com/cockroachdb/pq.(*rows).Next(0xc820b8a000, 0xc820b8a1b0, 0x9, 0x9, 0x0, 0x0)
	/go/src/github.com/cockroachdb/pq/conn.go:1541 +0x7a6
database/sql.(*Rows).Next(0xc820bb0000, 0xc82051c130)
--
	/go/src/github.com/cockroachdb/cockroach/sql/trace_test.go:98 +0x3e7
testing.tRunner(0xc820432090, 0x2308918)
	/usr/local/go/src/testing/testing.go:473 +0x98
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:582 +0x892
FAIL	github.com/cockroachdb/cockroach/sql	49.234s
?   	github.com/cockroachdb/cockroach/sql/driver	[no test files]
=== RUN   TestEval
--- PASS: TestEval (0.01s)
=== RUN   TestEvalError
--- PASS: TestEvalError (0.00s)
=== RUN   TestEvalComparisonExprCaching
--- PASS: TestEvalComparisonExprCaching (0.00s)
=== RUN   TestSimilarEscape
--- PASS: TestSimilarEscape (0.00s)
=== RUN   TestQualifiedNameString

```
Please assign, take a look and update the issue accordingly.

**Comment by tamird** [comment]
dupe #4965

**Comment by tamird** [comment]
wait, not a dupe.

**Comment by dt** [comment]
fixed by #4998
